cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocRuntimeJobManager.java

```diff
@@ -314,11 +315,15 @@ private LocalFile uploadFile(String bucket, String targetFilePath,
     BlobId blobId = BlobId.of(bucket, targetFilePath);
     BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("application/octet-stream").build();
 
+    Bucket bucketObj = getStorageClient().get(bucket);
+    LOG.debug("Uploading a file of size {} bytes from {} to gs://{}/{} of Location type : {} and Location : {}. ",
+              localFile.getSize(), localFile.getURI(), bucket, targetFilePath, bucketObj.getLocationType(),
+              bucketObj.getLocation());
```

Comment:
The bucket locationtype and location seems redundant information.

Reply:
Actually it differed for region vs multi region 
Like : 
`Location type : region and Location : US-EAST1.` 
and
`Location type : multi-region and Location : US`

Is having only `location` good enough to identify if it's regional or multi-regional ?